.github/workflows/testing.yml

```diff
@@ -35,6 +35,37 @@ jobs:
           coverage report -m
           bash <(curl -s https://codecov.io/bash)
 
+  test_check_unapplied_migrations_middleware:
+    name: test CheckUnappliedMigrationsMiddleware
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install Python dependencies
+      run: |
+        sudo apt-get install libkrb5-dev gettext
+        pip install -r requirements/devel.txt
+        pushd tcms/ && npm install && popd
+
+    - name: Run test
+      run: |
+        export LANG=en-us
+        ./manage.py migrate --settings=$(DJANGO_SETTINGS_MODULE) auth
+        ./manage.py migrate --settings=$(DJANGO_SETTINGS_MODULE) sites
+        ./manage.py migrate --settings=$(DJANGO_SETTINGS_MODULE) sessions
+
+        export TEST_CHECK_UNAPPLIED_MIGRATIONS_MIDDLEWARE=1
+        make test
```

Comment:
This wants to execute all possible tests which will fail because 90% of the DB tables are missing.  See how we run the tests in `integration_bugtracker.yml` for hints how to tell the test runner to only execute the middleware test.